java/src/client/facade/FacadeManager.java

```diff
@@ -0,0 +1,65 @@
+package client.facade;
+
+import shared.models.game.ClientModel;
+
+/**
+ * Created by audakel on 9/17/16.
+ */
+public class FacadeManager {
+    /** A reference to the game info in the client model, will update frequently */
+    private ClientModel mClientModel;
```

Comment:
There is no reason to preface member variables with anything. Just use `clientModel` or `model` (we can see it's a `ClientModel` by the type.

- Classes should be small enough that the variables do not become a burden.
- Any IDE we'll be using automatically highlights member variables (purple in IntelliJ's Darkula).